helpers-assets/js/translation-discussion.js

```diff
@@ -25,7 +25,8 @@ jQuery( function( $ ) {
 			post: $commentform.attr('id').split( '-' )[ 1 ],
 			meta: {
 				translation_id : $commentform.find('input[name=translation_id]').val(),
-				locale         : $commentform.find('input[name=comment_locale]').val()
+				locale         : $commentform.find('input[name=comment_locale]').val(),
```

Comment:
It's ok to proceed with adding this particular logic yet but it'd be good practice to already leave a TODO comment in this PR, or continue with this PR to add that piece.
```suggestion
				locale         : $commentform.find('input[name=comment_locale]').val(), // TODO change this based on the topic
```